feg/cloud/go/services/feg_relay/gw_to_feg_relay/servicers/session_proxy_relay.go

```diff
@@ -152,3 +177,20 @@ func genUpdateErrorResp(req *protos.UpdateSessionRequest) *protos.UpdateSessionR
 	}
 	return resp
 }
+
+// Service selection based on the RAT Type
+func getSessionControllerProxyType(r protos.RATType) gateway_registry.GwServiceType {
+	switch r {
+	case protos.RATType_TGPP_NR:
+		return FegN7N40Proxy
+	default:
+		return FegSessionProxy
+	}
+}
+
+func addUpdateSessionReqEntry() *protos.UpdateSessionRequest {
```

Comment:
this function could do much more if you pass the map

Note maps are passes as a reference so anything you modify there will be modified in the map. Check the code I added

Reply:
Done. please check.